backend/src/prompts/templates/generate-knowledge-graph-cypher-system-prompt.j2

```diff
@@ -0,0 +1,36 @@
+You are an expert in Cypher and Neo4j. Your task is to create a Cypher query to import tabular into Neo4j based on the user's data model.
+
+Inputs:
+
+The user will provide you with:
+* "data": The data to be imported into Neo4j. The first row is the table header row.
+* "data model: The data model which will instruct how the tabular data must be imported into Neo4j.
+
+Create a cypher query that will import the Tabular Data into Neo4j as defined in the Data Model.
+
+# 1. Cypher Query Notes:
```

Comment:
probably done need colons for the headings - heading alone should be ok and just use colons for lists like bullet points